pages/board/[id].tsx

```diff
@@ -0,0 +1,121 @@
+import { getArticle, getComments } from "@/lib/articleApi";
+import { useRouter } from "next/router";
+import React, { useEffect, useState } from "react";
+import kebabIcon from "@/public/images/icons/ic_kebab.svg";
+import Image from "next/image";
+import { Article, CommentsResponse } from "@/types/types";
+import styles from "./board.module.css";
+import profileIcon from "@/public/images/icons/ic_profile.svg";
+import heartIcon from "@/public/images/icons/ic_heart.svg";
+import backIcon from "@/public/images/icons/ic_back.svg";
+import Link from "next/link";
+
+const DetailBoard = () => {
+  const router = useRouter();
+  const { id } = router.query;
+  const [article, setArticle] = useState<Article>();
+  const [comments, setComments] = useState<CommentsResponse>();
+
+  const fetchArticle = async (articleId: number) => {
+    const result = await getArticle(articleId);
+    setArticle(result);
+  };
+
+  useEffect(() => {
+    id && fetchArticle(Number(id));
+  }, [id]);
+
+  const fetchComments = async (articleId: number, limit: number) => {
+    const result = await getComments(articleId, limit);
+    setComments(result);
+  };
+
+  useEffect(() => {
+    id && fetchComments(Number(id), 5);
+  }, [id]);
```

Comment:
```suggestion
  useEffect(() => {
  if(id){
  fetchArticle(Number(id));
  fetchComments(Number(id), 5);
  }
  }, [id]);
```
같은 조건에 동작하는 useEffect인것같아요. 하나로 통합해보는건 어떨까요? useEffect 둘 다 UI 렌더 이후에 동작해서 같은 조건을 보고 함수 호출하는 역할만 할꺼라 합쳐도 무방하고, 코드 보는게 좀더 깔끔해질것같아요. :-)